releases/release-1.31/release-notes/maps/pr-123597-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 123597
+releasenote:
+  text: Added an extra line between two different key value pairs under data when
+    running kubectl describe configmap.
```

What do you think about adding backticks here? 

```suggestion
    running `kubectl describe configmap`.
```